src/text-input-autocomplete.directive.ts

```diff
@@ -158,17 +162,27 @@ export class TextInputAutocompleteDirective implements OnDestroy {
         ),
         triggerCharacterPosition: this.elm.nativeElement.selectionStart
       };
+      //Get DOM element from component
+      this.menuElem = (this.menu.component.hostView as EmbeddedViewRef<any>)
```

Comment:
You can access the menu element by doing `this.menu.component.location.nativeElement`